### PR TITLE
Feature/590 update dep to polkadot v0.9.22

### PR DESCRIPTION
**Pull request by BillyWooo**
Upgrade polkadot to v0.9.22:
- bump up version 
- adapt API based on parity/cumulus upgrade

### CODE REVIEW

File: runtime/litentry/src/lib.rs [hunk 1]
```diff
@@ -435,7 +435,7 @@ impl pallet_utility::Config for Runtime {
 	type Event = Event;
 	type Call = Call;
 	type PalletsOrigin = OriginCaller;
-	type WeightInfo = weights::pallet_utility::WeightInfo<Runtime>;
+	type WeightInfo = ();
```
Comment (Kailai-Wang) [inline runtime/litentry/src/lib.rs:438]: This change can be reverted now, as the benchmark files were updated, please be sure to correct all 4 runtimes when you make changes

File: runtime/litentry/src/lib.rs [hunk 2]
```diff
@@ -612,7 +612,7 @@ impl pallet_treasury::Config for Runtime {
 	type Burn = Burn;
 	type BurnDestination = ();
 	type SpendFunds = ();
-	type WeightInfo = weights::pallet_treasury::WeightInfo<Runtime>;
+	type WeightInfo = ();
```
Comment (Kailai-Wang) [inline runtime/litentry/src/lib.rs:615]: see above

File: runtime/litentry/src/weights/mod.rs [hunk 3]
```diff
@@ -30,5 +30,5 @@ pub mod pallet_proxy;
 pub mod pallet_scheduler;
 pub mod pallet_session;
 pub mod pallet_timestamp;
-pub mod pallet_treasury;
-pub mod pallet_utility;
+// pub mod pallet_treasury;
+// pub mod pallet_utility;
```
Comment (Kailai-Wang) [inline runtime/litentry/src/weights/mod.rs:34]: see above

**Review by Kailai-Wang — CHANGES_REQUESTED** [review]

**Comment by Kailai-Wang** [comment]
btw could you bump the version in https://github.com/litentry/litentry-parachain/blob/04a30cde55b82e187bf13756ef555d9a12882fe8/pallets/identity-management/Cargo.toml too?

**Comment by BillyWooo** [comment]
> btw could you bump the version in https://github.com/litentry/litentry-parachain/blob/04a30cde55b82e187bf13756ef555d9a12882fe8/pallets/identity-management/Cargo.toml too?

Bumped. Done.

File: runtime/litentry/src/lib.rs (hunk 1 above)
Comment (BillyWooo) [reply runtime/litentry/src/lib.rs:438]: Yes. For the moment, only Litmus is supported. The rest three would be updated in further upgrade.

File: runtime/litentry/src/lib.rs (hunk 2 above)
Comment (BillyWooo) [reply runtime/litentry/src/lib.rs:615]: see above.

File: runtime/litentry/src/weights/mod.rs (hunk 3 above)
Comment (BillyWooo) [reply runtime/litentry/src/weights/mod.rs:34]: see above.

**Review by Kailai-Wang — APPROVED** [review]
Looks good

**Pull request merged**